django_orm_views/not_managed_model.py

```diff
@@ -0,0 +1,46 @@
+import copy
+
+from django.db.models import Model
+
+from .constants import SUB_SCHEMA_NAME
+
+
+class NotManagedModel(Model):
+    """
+    Abstract model used to define not managed django models that use
+    database views as a data source.
+
+    Warning: adding custom Meta class to the child model leads to an error
+    """
+    name = None
+
+    class Meta:
+        abstract = True
+
+    def __init_subclass__(cls, should_init=True):
+        """
+        Adds metadata to the class that makes it a not managed model with the data source equal to the
+        database view name generated by BasePostgresView parent class
+        """
+        super().__init_subclass__()
+
+        if should_init:
```

Comment:
I'm not sure what `should_init` is doing here at a glance - could we:
* Think about the variable name
* Consider adding docs to describe the reasoning for it

Reply:
Done